services/asset/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "asset"
-version = "0.4.0-beta2"
+version = "0.4.0-Alpha"
```

Comment:
It's rare to see someone mark a prerelease with an `Alpha`, `alpha` might be more appropriate

Reply:
alpha.0